agreement.md

```diff
@@ -0,0 +1,151 @@
+# Aragon Network Agreement
+
+## Section 0. Background
+The Aragon Network is a Decentralized Autonomous Organization that is building a dispute resolution protocol and other smart contract infrastructure.
+
+The Aragon Network is governed by Aragon Network Token (ANT) holders, who supported the initial creation of the network and under this Agreement will continue to guide its evolution through established governance processes.
+
+This Agreement exists to enumerate the shared mission, principles, and governance guidelines that bind ANT holders in their collaborative effort to create value for the Aragon Network.
+
+## Section 1. RFC 2119 compliance
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+## Section 2. Immutable guidelines
+The guidelines in this section SHALL NOT be amended. If any other guidelines in this Agreement conflict with these immutable guidelines, the immutable guidelines SHALL take priority for enforcement purposes. These guidelines apply to all proposals submitted to the Aragon Network organization.
+
+1. Proposals MUST be governed by this Agreement, or a future version of this Agreement as modified within the bounds of the Agreement.
+
+2. Disputes over a proposal relying on this Agreement MUST be resolved by Aragon Court.
+
+3. Proposals MUST NOT disproportionately benefit a majority of ANT holders voting on a proposal over the minority.
+
+4. Proposals MUST NOT attempt to cast a vote using ANT held in the address of any versions of the Agent app installed on the Aragon Network organization, or using ANT held in any Network contract that fulfills similar functionality as the Agent app.
+
+5. Proposals to amend either the current Aragon Court Controller contract or any new Controller contract that may be proposed for Aragon Court MUST NOT amend Aragon Court configurations, funds, and modules governors to be any entity other than an Agent app installed on the Aragon Network organization.
+
+6. If a proposal deploys a new Aragon Court Jurors Registry contract and/or amends Aragon Court to use a new Jurors Registry, then the new Jurors Registry MUST use ANJ as the native juror token.
+
+7. Proposals MUST NOT activate in Aragon Court any ANJ controlled by the Aragon Network.
+
+8. Proposals MUST NOT use the Aragon Network organization’s role as Aragon Court funds governor to transfer ANJ staked and/or activated by Aragon Court jurors to another Ethereum account.
+
+## Section 3. Mutable guidelines
+The guidelines in this section MAY be amended. These guidelines apply to all proposals put forth to the Aragon Network organization.
+
+1. Proposals MUST be consistent with the Aragon Manifesto.
```

Comment:
I think the Manifesto must be immutable. If someone wants to change it, they can launch their own fork of the Aragon Network.

Reply:
I think it should be possible to change the Manifesto. That doesn't mean it should be easy or happen often, and I don't expect anyone wanting to change it any time soon. But I do think it should be possible.

Reply:
>I think it should be possible to change the Manifesto. That doesn't mean it should be easy or happen often, and I don't expect anyone wanting to change it any time soon. But I do think it should be possible.

Why should it be possible?

The Manifesto states the values of the Aragon community. If someone disagrees with these values, for example if they think that centralized control and surveillance is good, then they should go and create their own community instead of trying to capture Aragon Network.

Reply:
I think it should be immutable for the same reason constitutions are immutable - it gives the network constituency actual full sovereignty. I'm not sure saying "if you don't accept the (immutability of the) Manifesto, you should leave and create your own network" is the best way to go about it. I also doubt that future changes to the Manifesto will include an endorsement of surveillance. It's much more likely that the community will want to positively expand the vision and value set, or perhaps rephrase/clarify something, simply because people learn and cultures evolve.